net-misc/astmanproxy/astmanproxy-1.28.2.ebuild

```diff
@@ -36,14 +36,11 @@ src_compile() {
 src_install() {
 	dosbin astmanproxy
 
-	dodoc README VERSIONS
-
-	docinto samples
-	dodoc samples/*
+	dodoc -r samples
+	einstalldocs
```

Comment:
I assume `VERSIONS` is just a dud?

Reply:
It's something like changelog, doesn't get picked up with `einstalldocs`. Changed it to previous version